src/plugins/demo-app/layout/Content.tsx

```diff
@@ -3,15 +3,15 @@ import { Layout } from 'antd'
 import styled from 'styled-components'
 import { useParams } from 'react-router-dom'
 
-import { TContent, DocumentPath } from '@development-framework/dm-core'
+import { DocumentPath } from '@development-framework/dm-core'
 
 const { Content } = Layout
 
 const PageContent = styled.div`
   padding: 20px;
 `
 
-const ContentWrapper = (props: TContent): JSX.Element => {
+const ContentWrapper = (props: any): JSX.Element => {
```

Comment:
```suggestion
const ContentWrapper = (props: any): JSX.Element => {
```
Can we use a type here?

Reply:
fixed